fhir/src/main/java/gov/cms/ab2d/fhir/FhirVersion.java

```diff
@@ -56,6 +56,10 @@ public static FhirVersion fromUrl(String url) {
                 .findFirst().orElse(null);
     }
 
+    public String getVersionString() {
+        return versionString;
+    }
```

Comment:
Technically, this is our version string so we have to make sure we mirror BFD if we make this change. BFD may not have a new FHIR version for each version of their endpoint.

Reply:
Made a change to separate the two out explicitly